filesystem/azure.go

```diff
@@ -212,7 +212,7 @@ func (f *AzureFile) Write(b []byte) (int, error) {
 		return 0, err
 	}
 	if err == io.EOF {
-		return 0, fmt.Errorf("Need blob content")
+		return 0, fmt.Errorf("Need write content")
```

Comment:
If the buffer doesn't have at least chunk size, the first read op doesn't return io.EOF, because it read something.